### PR TITLE
Pull request for Library App project.

### PR DESCRIPTION
**Pull request by FdI96**
Kindly request a code review.

This app shows different information dynamically by using Javascript.
It has a navbar that can select between Home, Menu and Contact information.
I used ES6 new features.

Home information contains a restaurant's picture and a brief description.
Menu information contains the restaurant's menu.
Contact is a section to contact the restaurant's owner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Restaurant App
+
+It's a Restaurant app to show the restaurant, the menu and to contact the owner information dynamically by clicking in the button in the Nav Bar
+
+![screenshot](./assets/screenshot/screenshot.jpg)
```
Comment (rukundoeric) [inline README.md]: - [x] [MANDATORY] This image link is invalid, kindly fix it. 
       Screen Shot 2021-06-21 at 02 06 00

File: README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- Has a contact section.
+
+## Built With
+
+- Html, CSS, and JavaScript
+- Bootstrap
+
+## Live Demo
+
+[Live Demo Link](https://fdi96.github.io/Restaurant-App/)
```
Comment (rukundoeric) [inline README.md:20]: - [x] [MANDATORY] This live-demo link is returning page not found. please, make sure the live-demo link is valid.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Install
+
+You can download the code to your personal computer clonig the repository using the following command
+
+```
+git clone
```
Comment (rukundoeric) [inline README.md]: - [x] [MANDATORY] It would be better to complete this command to avoid confusion.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
# Status: ♻️ CHANGES REQUIRED ♻️
_Great job guys so far 👏 💯 👍. However, there are few issues we need to address before we move on, please check the comments below._

- ### To Highlight
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯 


- ### To Improve
  - Check the comments below....

_Please make all the required changes before you submit another code review. If you have any problems or concerns, you can reach out to me on Slack: Rukundo Eric 🚀 . I wish you all the very best 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), 
Slack: Rukundo Eric**

File: README.md (hunk 3 above)
Comment (FdI96) [reply README.md]: Done! Thanks for the review.

File: README.md (hunk 2 above)
Comment (FdI96) [reply README.md:20]: Done! Thanks for the review.

File: README.md (hunk 1 above)
Comment (FdI96) [reply README.md]: Done! Thanks for the review.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import navBarComponent from './navBarElement';
+import contentHomeComponent from './home';
+import contentMenuComponent from './menu';
+import contentContactComponent from './contact';
+
+console.log();
```
Comment (rindrajosia) [inline src/index.js]: No console.log commited to repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- App is deployed :+1: 

To improve :factory_worker: [REQUIRED]:

- [x] The image on the home page of your app is broken when I open the live demo.


Nice to have [OPTIONAL] :

- [x] Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
- [x] No console.log commited to repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/console_log_requirement.png) from Pathwright)


That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @FdI96**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**